tests/unit/IonBinaryWriterTest.js

```diff
@@ -625,10 +625,12 @@ define([
       ]);
 
     // Symbols
-
     writerTest('Writes null symbol by detecting null',
       (writer) => { writer.writeSymbol(null) },
         [0x7f]);
+    writerTest('Writes null symbol by detecting null and allows user to pass null into set annotations',
```

Comment:
nope

Reply:
If you disagree with a PR comment explain why or if need clarification say so. Answering "nope" is extremely unhelpful

Reply:
sorry i meant were gonna throw on null instead as the apis no longer return null.

Reply:
ok. In any case the comment is to use a test description that describes the expected behavior and not what the test is doing. 

When you change to throw on `null` update the description with the expected behavior

Reply:
gotcha, also the reason i dont plan on getting into parameterization is that were going to delete all these tests for mocha shortly.

Reply:
Ok that makes sense. As we talked offline if you can write the mocha tests now might as well do it, but since the test framework migration is in flight I'm ok waiting until mocha is fully integrated to migrate and reduce the duplication 

I'll let you decide what's the best path here